packages/contracts/package.json

```diff
@@ -49,7 +49,8 @@
     "dollar",
     "decentralization",
     "token",
-    "algorithmic"
+    "algorithmic",
+    "solidity"
```

Comment:
Solidity is capitalized, perhaps it is proper to remove this?

Reply:
yeah will remove this, is not part of the integration anyway

Reply:
I also think this can be removed